wiki/osu!stream/de.md

```diff
@@ -0,0 +1,254 @@
+
+## Hinzufügen von Beatmaps
+
+**Stellen Sie sicher, dass das iDevice eine Verbindung zum Internet herstellen kann (entweder über Wi-Fi oder 3G / 4G).**
+
+- Öffnen Sie den osu! -Stream.
+- Drücken Sie die osu! Symbol und es sollte zum Hauptmenü geleitet werden (oder Tutorial, wenn das erste Mal)
+- Drücken Sie die Leiste "Store"
+  - Alternativ gehen Sie zu "Play" und drücken Sie "Download more songs!"
+- Wählen Sie eine gewünschte Packung.
+  - Es wird empfohlen, zuerst Pakete herunterzuladen, die kein Geld kosten. Gratis-Packs haben anstelle des "USD $" -Kosten-Tags "FREE".
+
+## Was ist der Unterschied?
+
+- **Nur auf iOS-Geräten spielbar.** (vorerst)
+- Beatmaps werden von Grund auf neu erstellt.
+- Zum Herunterladen der Song-Packs ist kein Konto erforderlich.
+- Kann osu! Stream mit einem Twitter-Account integrieren.
+- Der Highscore kann anonym unter dem Namen "Gast" an die Site gesendet werden (falls dies vorkommt, jedoch eine Internetverbindung erforderlich ist).
+- Die Kreisuhr wird ausgetauscht und mit einer kleinen Randlinie nach unten verschoben, die je nach Genauigkeit farbig ist. \ [Bottom (Long)\]
+- **Durchschnittliches Timing** ist auf der Notenseite angegeben.
+- Kann weder alte Datensätze noch Wiederholungen speichern.
+- Skin kann nicht geändert werden und Modifikationen können (noch) nicht angewendet werden.
+- Ostereier können gefunden werden, aber es gibt keine Seite mit Erfolgen ...
+
+### Neues Spielsystem
+
+**Hinweis:** Testen Sie die neuen Gameplay-Elemente im "Tutorial" -Bildschirm.
+
+#### Finger Guides Anzeige
+
+![Finger Guides anzeigen][Os-FG.jpg]
+
+_Finger Guides display_
+
+- Der zwei durchsichtige Kreis, der im Spiel angezeigt wird, soll Anfängern helfen, zu wissen, wann sie die Objekte treffen müssen.
+  - Es zeigt die vorgeschlagene Methode, um die de design der Karte mit Perfektion.
+- Die grüne Farbe ist für die linke und die rote für die rechte Hand.
+- Diese Funktion kann aus dem Optionsmenü oder nach Abschluss des Tutorials deaktiviert werden.
+
+######Connected Beats
+
+![Connected Beats][OS-CB.png]
+
+_Connected Beats_
+
+- Verbundene Beats, wie der Titel vorschlug, erfordert, dass der Spieler beide Objekte gleichzeitig drückt.
+- Verbundene Beats sind nicht nur auf "Hit Circle-Hit Circle" beschränkt, sondern können "Hit Circle-Slider", "Hold Circle-Slider" usw. sein.
+- Verbundene Beats können identifiziert werden, indem eine gerade weiße Linie mit einem anderen Objekt verbunden wird.
+
+##### Hold circle
+
+![Hold circle 1][PC.jpg]
+
+_Hold circle 1_
+
+-
+
+![Hold circle 2][OS-HC2.jpg]
+
+_Hold circle 2_
+
+- Halte den Kreis so lange, bis der Kreis "explodiert
+- 300 wenn perfekt gemacht.
+  - 100 für einen ungenaueren anfänglichen Halt und ein korrektes Halten.
+  - Zumindest 50 für das Berühren.
+  - 30 für jede Explosion.
+  - Multiplikator-Reset, wenn die Kreise nicht richtig gehalten wurden, und Miss, wenn sie dies nicht taten.
+

```

Comment:
It looks like a bunch of content is missing around here.

From `New Modes to replace the difficulty` to half of `Score`

Reply:
Tried to fix. Please let me know of any errors.